docs/typescript/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@mart3323/jsmacros-types",
```

Comment:
I have grabbed the `@jsmacros` org. want me to add you to it?

Reply:
Up to you - though keep in mind that as soon as we switch to the final package name, anything published (including from this unmerged MR) will become locked in. NPM does not allow editing/republishing to the same combination of package name and version number
source: https://docs.npmjs.com/policies/unpublish